newdle_frontend.js

```diff
@@ -43,19 +44,15 @@ function onSubmit(){
     winningConditions()
 }
 
+
 // Functions otherwise
 
-GUESS_BOX.addEventListener("keypress", function(event){
-        if (event.key === "Enter"){
-            onSubmit();
-        }
-    }
-);
 
 function getHeadlines(headlines_json){
-    document.getElementById("presented_headline").innerText = headlines_json.scrambled_headline
-    headlineArray = headlines_json.headline.split(" ")
-    headlineArrayLower = headlines_json.headline.toLowerCase().split(" ")
+    console.log(headlines_json)
+    document.getElementById("presented_headline").innerText = "Transfer news LIVE: Barcelona 'preparing' Lewandowski dib with Arsenal, Man Utd and Liverpool latest - The Mirror"
```

Comment:
Have you left in a permentant headline for testing here?

Reply:
It also starts completely solved?

Reply:
That's weird, I put this back afterwards but seem to have pushed before I reverted the testing coding?